### PR TITLE
Zenhub board style updates

### PR DESCRIPTION
**Pull request by olets**
Addresses a few lingering Zenhub styling issues. (Unlike #754, which was darkening things GitHub-Dark had missed, this PR is for polishing up things that were already dark, including bringing the GitHub-Dark Zenhub styles in line with the latest default Zenhub styles)

- Zenhub nav background matches page
- Issue label section background matches issue title section background
- Pipeline issue count is bright and story point count is dim
- Pipeline control icons color match theme
- Pipeline heading background matches pipeline background
- Updates "New Issue" button to follow new flat Zenhub native styles
  (invert native hover/no-hover colors)
- Updates Zenhub nav buttons' background to follow new flat Zenhub native styles
- Search bar text matches theme
- Lightens pipeline background (including scroll track)

### Default Zenhub

Notably now with flat buttons

screen shot 2018-10-28 at 15 30 51

### Before this PR

Not pictured: text entered in search box was black

screen shot 2018-10-28 at 17 08 31

### With this PR

screen shot 2018-10-28 at 16 48 36

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -4072,9 +4079,26 @@
   .zhc-btn--primary-bordered:hover, .zhc-btn--secondary-bordered:hover,
   .zhc-btn--secondary:hover, .zhc-issue-card__actions__btn:hover,
   .zhc-btn:hover .zhc-btn--text-secondary:hover, .zhc-filter-expand-button:hover,
-  .zhc-chart-legend__labels--notselected {
+  .zhc-chart-legend__labels--notselected, .zhc-pipeline-header,
+  .zhc-pipeline ::-webkit-scrollbar-track, ::-webkit-scrollbar-corner,
+  .zhc-pipeline-body { {
```
Comment (Mottie) [inline github-dark.css]: Please remove the extra curly bracket here. Thanks!

Comment (olets) [reply github-dark.css]: Sorry about that!

File: github-dark.css [hunk 2]
```diff
@@ -4036,7 +4043,7 @@
     background: #181818 !important;
     border-color: #484848 !important;
   }
-  .zhc-issue-card__meta, .zhc-selection-filter, .zhc-tooltip-action__item,
+  .zhc-selection-filter, .zhc-tooltip-action__item,
   .zh-milestone-dates, .zh-epic-issue-list-footer, .zh-select-menu-header,
```
[the remark this answers is not in the document]
Comment (olets) [reply github-dark.css]: Yes indeed. I have the fix ready. Do you have a preference for whether I amend or add a new bugfix commit @Mottie?

[the remark this answers is not in the document]
Comment (Mottie) [reply github-dark.css]: Commit it here and we'll squash it all in one go😸

**Comment by olets** [comment]
Added a commit with those fixes, realized I hadn't actually succeeded in dropping the Zenhub nav's background 😮

This is what I'm seeing now, with no customizations:

<kbd>screen shot 2018-10-29 at 13 18 06</kbd>

**Comment by silverwind** [comment]
Just going by your last screenshot, I'd suggest not completely flattening the issue boxes and use our existing color shades more. If there is a border in the original, it should be visible, if just slightly.

Suggesting you use colors similar as seen on GitHub's projects, with a `#202020` header:

screenshot 2018-10-30 at 17 46 12

**Comment by olets** [comment]
@silverwind I agree that it would be nice to update things to match Projects. Didn't want to take on anything as potentially controversial as that here, just wanted the pipelines to not look as rough. I'd be happy to swing this PR in that direction if people are into it

**Comment by silverwind** [comment]
We can do that in another PR too I suppose. @Mottie can you verify the changes? You know I don't use Zenhub personally.

**Comment by Mottie** [comment]
Yes, please make it purdy in another PR 😸

**Pull request merged**